doc/external/pytorch_tutorials_hyperparameter_tuning_tutorial.py

```diff
@@ -42,13 +42,15 @@
 """
 from functools import partial
 import os
+from tempfile import TemporaryDirectory
 import torch
 import torch.nn as nn
 import torch.nn.functional as F
 import torch.optim as optim
 from torch.utils.data import random_split
 import torchvision
 import torchvision.transforms as transforms
+import ray
```

Comment:
Do we need this? Or should this be `import ray.train`

Reply:
yeah, or just use `train` only. Let me remove it